csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1647,7 +1651,7 @@ Valid values are:
     not_contacted
     open
 
-Summary of involvement (`summary`) of value type `string` with 1 or more characters contains additional context regarding what is going on.
+Summary of involvement (`summary`) of value type `string` with 1 or more characters contains optional additional context regarding what is going on.
```

Comment:
That `summary` is optional was already mentioned in line 1621. We should not double that.